doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ppr.rst

```diff
@@ -0,0 +1,175 @@
+.. _ug_nrf54h20_ppr:
+
+Working with the PPR core
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   The PPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *Peripheral Processor* (PPR).
+The PPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API.
+These peripherals have IRQs routed to PPR:
+
+* ##TODO
+
+All other peripherals available to the application core can also be used with PPR.
+However, they require the use of *polling mode*.
+
+.. _vpr_ppr_nrf54h20_initiating:
+
+Using Zephyr multithreaded mode on PPR
+**************************************
+
+The PPR core can operate as a general-purpose core, running under the full Zephyr kernel.
+Building the PPR target is similar to building the application core, but the application core build must include an overlay that enables the PPR core.
+
+Bootstrapping the PPR core
+==========================
+
+The |NCS| provides a PPR snippet that adds the overlay needed for bootstrapping the PPR core.
+The primary purpose of this snippet is to enable the transfer of the PPR code to the designated memory region (if required) and initiate the PPR core.
+
+When building for the ``nrf54h20dk/nrf54h20/cpuppr`` target, a minimal sample is automatically loaded onto the application core.
+For more details, see :ref:`building_nrf54h_app_ppr_core`.
+
+Memory allocation
+*****************
+
+Running the PPR CPU can lead to increased latency when accessing ``RAM_30``.
+To mitigate this, you should use ``RAM_30`` exclusively for PPR code, PPR data, and non-time-sensitive information from the application CPU.
+For data that requires strict access times, such as CPU data used in low-latency ISRs, you should use local RAM or, when greater latency is acceptable, ``RAM_0x``.
+The DMA buffers should be placed in memory designed to a given peripheral.
```

Comment:
We could add that if a RAM_30 and RAM_31 are designed, RAM_31 should be selected to avoid memory access latency caused by PPR